Dockerfile

```diff
@@ -65,8 +65,7 @@ RUN apk --no-cache --quiet add \
   bash \
   build-base \
   curl \
-  git \
-  python
```

Comment:
python2 package is [no longer provided in Alpine](https://git.alpinelinux.org/aports/commit/?id=5ad0ec7d) (as of 3.12) (python3 is available).
Not clear to me why this was added but it may be OK to get rid of it now. [Old commit](https://github.com/artsy/force/blob/caf152aabf79d6f3cbe5d3f333becc8eb376371e/Dockerfile#L4) that added python.